server/app/services/database.service.ts

```diff
@@ -169,4 +169,46 @@ export class DatabaseService {
         const values = [eventKey];
         return this.pool.query(query, values);
     }
+
+    async updateQuiz(quizId: string, maxDuration: number, title: string): Promise<QueryResult> {
```

Comment:
While doing updates, waiting for result to complete can cause UI to block.

Reply:
This sounds like a problem that should be dealth with asynchronously in the front-end. The UI could have some sort of waiting/loading bar to indicate the user that there is an undergoing operation. Or the user could get a small notification when the database query is done, if pertinent